__tests__/app.test.js

```diff
@@ -76,3 +76,41 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: returns an article to the client", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(Object.keys(article)).toEqual(
+          expect.arrayContaining([
+            "article_id",
+            "author",
+            "title",
+            "body",
+            "topic",
+            "created_at",
+            "votes",
```

Comment:
Since we know exactly what article we are getting back, we can be more specific.  Add the exact values for the object we expect back